src/app/payments/send-lightning.ts

```diff
@@ -322,11 +333,13 @@ const executePaymentViaIntraledger = async <
   paymentFlow,
   senderWallet,
   senderUsername,
+  senderDisplayCurrency,
   memo,
 }: {
   paymentFlow: PaymentFlow<S, R>
   senderWallet: WalletDescriptor<S>
   senderUsername: Username | undefined
+  senderDisplayCurrency: DisplayCurrency
```

Comment:
these two fields are from account.. maybe is better to pass `senderAccount` as param

Reply:
Done in next PR https://github.com/GaloyMoney/galoy/pull/2384/commits/ba909150b62c16f839845e282e9b25bd4577721a